src/main/java/ladysnake/satin/mixin/client/render/RenderLayerMixin.java

```diff
@@ -0,0 +1,42 @@
+package ladysnake.satin.mixin.client.render;
+
+import com.google.common.collect.ImmutableList;
+import java.util.HashSet;
+import java.util.Set;
+import ladysnake.satin.impl.BlockRenderLayerRegistryImplementation;
+import net.minecraft.client.render.RenderLayer;
+import net.minecraft.client.render.RenderPhase;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(RenderLayer.class)
+public abstract class RenderLayerMixin extends RenderPhase{
+    private RenderLayerMixin(){
+        super(null, null, null);
+    }
+    
+    @Redirect(
+        method = "getBlockLayers",
+        at = @At(
+            value = "INVOKE",
+            target = "Lcom/google/common/collect/ImmutableList;of(Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)Lcom/google/common/collect/ImmutableList;"
+        )
+    )
```

Comment:
Not fond of the redirect. I would instead inject at return, then `cir.setReturnValue(ImmutableList.builder().addAll(cir.getReturnValue()).addAll(BlockRenderLayerRegistryImpl.INSTANCE.getLayers()))`